Frameworks/UIKit/UIScrollView.mm

```diff
@@ -56,6 +58,7 @@
 static const bool DEBUG_DELEGATE = DEBUG_ALL || false;
 static const bool DEBUG_INSETS = DEBUG_ALL || false;
 static const bool DEBUG_ZOOM = DEBUG_ALL || false;
+static const bool DEBUG_DMGesture = DEBUG_ALL || false;
```

Comment:
>DEBUG_DMGesture [](start = 18, length = 15)

nit: nobody will know what 'dm' is when they come by later.  Can we call this DEBUG_DMANIP_GESTURE?